tests/test_other.py

```diff
@@ -8186,7 +8186,17 @@ def test(args, closure, opt):
   # The no_windows/no_mac decorators also solve that problem.
   @no_windows("Code size is slightly different on Windows")
   @no_mac("Code size is slightly different on Mac")
```

Comment:
Oh hmm, when did these two decorators appear? Instead of this, can we add a Windows and Mac specific slop to the test? Otherwise this could regress without being detected.

Reply:
We should also really investigate what going on there.   Do you know if these differences appear in CI ?   I guess I can just upload a change removing these and see if CI fails...

Reply:
For the really small tests here I was also thinking that we might want to check-in the full pretty-printed JS and the full disassembled wasm.   That was we can in the diff exactly what changed when one of these tests fails.  

I often end up investigating regressions and wish I could just see the diff without having to `git checkout` back and forth between revisions of use side-by-side checkouts of emscripten (which I have not tried yet but I think might be good alternative to this suggestion).

It would certainly make them way more fragile, but honestly if you are making a change that effects the 100 bytes that makes up our smallest output.. we all should really be aware of those changes, including the author and reviewer of the change.